documentation/Restart-File.md

```diff
@@ -0,0 +1,64 @@
+# Experimental Feature: Restart File
+
+When using Trapperkeeper apps inside of packages, it is convenient for a service
+framework to have a clear indication as to when all of the Trapperkeeper
+services in the app have been started -- as opposed to just knowing when the
+Java process hosting the app has been spawned.  The "restart file" feature in
+Trapperkeeper provides this capability.  As a reference example, the
+[EZBake](https://github.com/puppetlabs/ezbake) build system for
+Trapperkeeper-based applications makes use of the "restart file" feature.  Its
```

Comment:
Maybe a more specific link would be helpful, not sure?  As it stands, we don't plan on making this an externally configurable feature in EzBake - more just some internal plumbing in the build scripts.  I'm not sure if throwing TK users into the middle of the build scripts without context would be too much more helpful.  I'd also be okay with just removing the EzBake comment altogether but @cprice404 thought there might be value just in mentioning it as an example use case.